src/custom/utils/metadata.ts

```diff
@@ -0,0 +1,51 @@
+export enum MetadataKind {
+  REFERRAL = 'referrer',
```

Comment:
should we use the same name? REFERRAL or REFERRER?

Reply:
Good question, I pick names based on the doc https://docs.google.com/document/d/1Zu-ygdI1QpsFrJp_Ux4Q5KQx6iQLRfVscjebaYB_RV0/edit#heading=h.o6tsb2ejbntu

Reply:
But shouldn't it match in the enum? `REFERRAL = 'referrer'`

Reply:
I removed the `MetadataKind` enum in favor of using the referrer key directly

```js
export type MetadataDoc = {
  referrer?: ReferralMetadata
}
```